### PR TITLE
Docs for projections

### PR DESCRIPTION
**Pull request by Dmytry-Dyachenko**
Add comments for projections.
Rename `CoverUrlView `-> `CoverView`.

### CODE REVIEW

File: model/src/main/proto/javaclasses/exlibris/q/projections.proto [hunk 1]
```diff
@@ -50,76 +47,66 @@ message BookDetailsView {
...
-    // URL for book cover.
+    // URL for the book cover.
     spine.net.Url book_cover_url = 2;
 
-    // Book’s title.
+    // The book title.
     BookTitle title = 3;
 
-    // Full author’s name.
+    // The list of author’s names.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/projections.proto]: names->name

**Review by alexanderkarpetc — CHANGES_REQUESTED** [review]
Make 1 small change plz

**Review by PaulAgeyev — APPROVED** [review]
LGTM

**Review by alexanderkarpetc — APPROVED** [review]
LGTM

**Pull request merged**